pages/about.html

```diff
@@ -5,8 +5,51 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link href="../style.css" rel="stylesheet" />
 </head>
 <body>
+
+  <header>
+    About Me
```

Comment:
This should be a `h1` (it's the top header of your site):

```suggestion
    <h1>About Me</h1>
```